lib/morse_translator.rb

```diff
@@ -38,6 +39,30 @@ def initialize
                     "9" => "----.",
                     "0" => "-----",
                     " " => " "}
+
+    @morse_to_eng_dictionary = @dictionary.invert
+
+  end
+
+  def translate(input)
+    input_array = input.downcase.chars
+    translate = []
+      input_array.map do |character|
+        translate << @dictionary[character]
```

Comment:
Here, you are using map like each, that is you are using it to iterate over each element, change that element, and store it in a container. Instead, you should use map like this:

```ruby
translate = input_array.map do |character|
  @dictionary[character]
end
```